modules/release-notes/pages/index.adoc

```diff
@@ -122,6 +122,28 @@ vertex function can also be used with vertex input parameters in the query-body
 
 == Fixed issues
 
+=== Fixed and Improved in 3.9.2-1
+
+==== Crashes and Deadlocks
+* Fixed a GPE crash caused by aborted query still being used by running thread.
+* Fixed crash issue after RESTPP auth refresh.
+* Fixed GSE crash due to the check failure when refreshing leader info after 60 minute timeout.
+* Fixed GPE crash and non-recovery due to Kafka retention to a higher offset.
```

Comment:
Please check the word "retention"

Reply:
Why was the delta topic deleted in some of nodes in the cluster? Is that a normal thing (e.g., it always gets deleted eventually) or is it unusual? "offset comparison misread" seems excessive. 

Don't overstress about this.  It should be correct-ish and concise.

Reply:
To simplify it I made it like this:

* Fixed GPE crash and non-recovery due to Kafka and GPE offset when related nodes were deleted.

...honestly not entirely sure what a "delta topic" is in Kafka.

Reply:
"related nodes" was changed to "adjacent vertices" as per our discussion. Thank you.